cordapp-upgrades/contracts/v1-contract/src/main/resources/META-INF/services/net.corda.core.serialization.SerializationWhitelist

```diff
@@ -0,0 +1 @@
+# Register here any CorDapp plugins extending from net.corda.node.CordaPluginRegistry.
```

Comment:
Let's get of this file as we're not using it. It's relevant for what this sample is trying to do.